### ISSUE TITLE
トレーニング終了後心拍数のゾーン毎パーセント提示できるようにする

### ISSUE THREAD

**Issue by copios** [copios-jp/red-falcon#9]
Record training session in memory
  1. Transmitter must have name/sex/weight/age/maxHR configured

  - "Camera" button is enabled when transmitter configuration is complete
    - Name
    - Age
    - Sex
    - Weight
    - HR Max

  - Clicking the camera starts recording
    - Stats are gathered each second (nanotimer)
    - The camera icon converts to Stop button
    - Transmitter configuration cannot be changed while recording
    - HR% graph will show in real time in footer of Transmitter card (last 300 samples - 5 min window)

  - Clicking the Stop button stops recording

  - "Report" button is enabled when there is captured training data

  - Clicking the Report button shows a report for the training session (**I am not sure there will be enough time to do this right**)
    - Stats
      - Name, Sex, Age, Weight, HR Max
      - Exercise duration
      - Average HR
      - Average HR zone (with explanation for zone)
      - Estimated Calorie expended 
      - Minami points 😄 
    - Zones
      - Pie chard of zone time
      - Bar chart of time in zone with values 
    - Timelapse
      - Line chart of %max over time for training

Stopping the application or clicking on the Camera Icon will result in the loss of training data.

**Comment by copios** [comment]
[MORGAN-RANDY-2018-12-30T18-49-52.822Z.pdf](https://github.com/copios-jp/red-falcon/files/2717461/MORGAN-RANDY-2018-12-30T18-49-52.822Z.pdf)

**Comment by copios** [comment]
![screen shot 2018-12-31 at 4 37 53 am](https://user-images.githubusercontent.com/44760707/50550535-df828e80-0cb5-11e9-8b16-cb3d164801ac.png)
